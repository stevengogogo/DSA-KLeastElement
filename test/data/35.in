10 50000
-8345 1621 -5286 -8753 -9101 5475 -9452 5584 -2912 9086
Insert 4 -8868
Insert 12 7942
Reverse 5 9
Query 6 6 1
Delete 11
Reverse 9 9
Query 1 5 2
Insert 8 -3996
Delete 8
Insert 8 -2653
Delete 8
Delete 11
Insert 8 -9383
Delete 4
Query 8 8 1
Reverse 3 10
Reverse 8 9
Query 7 8 2
Query 9 10 1
Delete 4
Reverse 2 6
Reverse 1 4
Reverse 7 9
Query 9 9 1
Reverse 9 9
Reverse 9 9
Query 3 8 4
Reverse 5 9
Query 3 4 1
Reverse 8 9
Query 5 6 1
Query 9 9 1
Delete 2
Reverse 3 4
Query 7 8 1
Delete 5
Delete 6
Insert 7 6418
Insert 7 1044
Query 1 7 6
Query 4 6 1
Reverse 3 4
Delete 2
Query 1 2 1
Delete 5
Insert 6 916
Delete 3
Insert 6 5364
Delete 2
Delete 4
Insert 2 2520
Delete 5
Delete 5
Delete 2
Query 3 3 1
Query 3 3 1
Reverse 2 3
Insert 4 5890
Reverse 2 4
Delete 2
Query 2 3 1
Query 2 3 2
Reverse 2 2
Delete 1
Query 1 2 2
Reverse 2 2
Insert 2 5874
Reverse 2 2
Insert 3 3000
Insert 1 -5661
Insert 6 2541
Query 6 6 1
Query 3 5 3
Reverse 3 5
Query 5 6 1
Query 1 6 2
Insert 1 -5733
Delete 2
Query 5 5 1
Insert 3 7284
Delete 6
Insert 5 8744
Query 4 4 1
Insert 4 6589
Query 1 4 2
Insert 7 4713
Query 9 9 1
Reverse 1 6
Reverse 1 1
Reverse 5 6
Query 5 5 1
Insert 7 -4254
Query 6 7 1
Insert 9 7550
Delete 10
Query 10 10 1
Query 8 8 1
Insert 2 4095
Insert 7 6809
Delete 10
Delete 5
Reverse 10 10
Delete 10
Insert 5 4951
Delete 5
Query 6 7 2
Query 9 9 1
Delete 3
Insert 5 -6702
Delete 3
Insert 3 -8779
Delete 4
Delete 7
Query 6 7 2
Delete 7
Query 6 6 1
Insert 6 -51
Query 7 7 1
Query 1 3 1
Reverse 1 4
Query 2 6 3
Query 2 5 1
Query 5 7 2
Insert 6 4059
Delete 5
Query 4 6 2
Query 4 4 1
Query 5 7 3
Insert 8 4657
Delete 6
Delete 4
Reverse 5 6
Delete 5
Delete 3
Delete 4
Reverse 3 3
Delete 1
Reverse 1 2
Delete 1
Insert 1 8495
Query 2 2 1
Query 1 1 1
Query 1 1 1
Reverse 2 2
Delete 1
Reverse 1 1
Insert 2 3081
Query 2 2 1
Reverse 1 2
Query 2 2 1
Insert 3 6537
Reverse 1 1
Reverse 2 3
Delete 2
Query 1 2 1
Insert 1 4444
Query 2 3 1
Query 2 2 1
Delete 2
Delete 2
Reverse 1 1
Query 1 1 1
Reverse 1 1
Reverse 1 1
Insert 1 -8081
Reverse 1 2
Insert 2 8846
Reverse 2 2
Query 2 3 1
Delete 3
Insert 1 2225
Reverse 3 3
Insert 4 7428
Insert 2 4034
Insert 4 6416
Query 5 6 1
Reverse 1 4
Delete 2
Delete 1
Query 3 3 1
Reverse 2 3
Reverse 2 2
Reverse 2 4
Reverse 3 3
Delete 2
Reverse 2 3
Insert 4 2806
Insert 4 -6793
Delete 1
Delete 3
Reverse 3 3
Delete 1
Query 2 2 1
Query 2 2 1
Delete 2
Insert 1 6815
Query 1 1 1
Reverse 2 2
Delete 1
Reverse 1 1
Query 1 1 1
Delete 1
Insert 1 -7587
Reverse 1 1
Reverse 1 1
Query 1 1 1
Insert 2 -2303
Query 1 1 1
Reverse 2 2
Delete 1
Query 1 1 1
Insert 2 1095
Query 2 2 1
Reverse 1 2
Query 2 2 1
Query 2 2 1
Delete 1
Query 1 1 1
Query 1 1 1
Query 1 1 1
Query 1 1 1
Delete 1
Insert 1 -4869
Insert 1 -9574
Delete 1
Reverse 1 1
Query 1 1 1
Insert 1 -605
Reverse 2 2
Delete 2
Query 1 1 1
Query 1 1 1
Reverse 1 1
Reverse 1 1
Query 1 1 1
Insert 1 1726
Query 1 2 1
Query 2 2 1
Delete 2
Delete 1
