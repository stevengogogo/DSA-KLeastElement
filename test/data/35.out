-9452
-8345
-9101
5584
-9452
5584
-2912
-8345
-9452
1621
-2912
5475
-8345
-9101
1044
1044
-5286
1044
1044
2541
5874
2541
-5286
5874
1044
-5286
2541
-5733
-5286
2541
4713
6809
7550
7550
-5286
-5286
-8779
4095
-8779
-51
4059
8744
4059
-8779
8495
8495
3081
-8779
-8779
-8779
3081
4444
-8081
7428
8846
2806
2806
6815
2225
-7587
-7587
-2303
1095
-2303
-2303
-2303
-2303
-2303
-2303
-4869
-605
-605
-605
-605
-605
